ui/src/app/services/logger.service.ts

```diff
@@ -4,11 +4,9 @@ import { isDevMode } from "@angular/core";
 
 @Injectable()
 export class LoggerService {
-  makeLog(logLocation: string,logValue: string) {
+  makeLog(logLocation: string, logValue: string) {
```

Comment:
We should consider swapping our logging over to [winston](https://github.com/winstonjs/winston).